Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR.yml

```
@@ -837,8 +837,8 @@ script:
       - 'false'
       required: false
       secret: false
-    deprecated: false
-    description: Isolates the specified endpoint. This command will be deprecated soon, use `xdr-endpoint-isolate` instead.
+    deprecated: true
+    description: This command is now deprecated, please use `xdr-endpoint-isolate` instead.
```

```suggestion
    description: Deprecated. Use `xdr-endpoint-isolate` instead.
```
In terms of UI, we want the customer to know straightaway that it is deprecated and what to do instead.
In all relevant places, please make the necessary change.